docs/tctl/admin/db/clean.md

```diff
@@ -0,0 +1,113 @@
+---
+id: clean
+title: tctl admin db clean
+description: Cleaning up corrupted Workflows
+tags:
+  - reference
+  - tctl
+  - admin
+  - db
+---
+
+The `tctl admin db clean` command cleans corrupted [Workflows](/docs/concepts/what-is-a-workflow) from the targeted database.
```

Comment:
Workflow Executions


And more specifically - Event Histories I think... can double check with @dtornow if we need to.